Xamarin.Forms.Platform.WPF/Properties/AssemblyInfo.cs

```diff
@@ -2,9 +2,32 @@
 using System.Runtime.InteropServices;
 using System.Windows;
 using Xamarin.Forms;
-using Xamarin.Forms.Platform.WPF;
+using Xamarin.Forms.Platform.WPF; 
+
+// L'affectation de la valeur false à ComVisible rend les types invisibles dans cet assembly
```

Comment:
What's with all the French? :D also, any idea where the extra attributes came from and if they are needed?

Reply:
I have no idea, I just left all the old stuff in there because I just wanted to do the changes for the new sdk format. Should I remove it?